From e613160ef171aaaffb40c0b800b6c6def110ee18 Mon Sep 17 00:00:00 2001
From: Gowtham Tammana <g-tammana@ti.com>
Date: Tue, 4 May 2021 22:08:25 -0500
Subject: [PATCH] gstdrm: Remove wayland as required dependency

gstdrmallocator is using dumb buffer api for buffer allocation and as
such there is no hard dependency on wayland. Remove it from required.

Upstream-Status: Inappropriate [depends on drmallocator patches in
meta-argo]

Signed-off-by: Gowtham Tammana <g-tammana@ti.com>
---
 gst-libs/gst/drm/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst-libs/gst/drm/meson.build b/gst-libs/gst/drm/meson.build
index 3310cc6..0700cab 100644
--- a/gst-libs/gst/drm/meson.build
+++ b/gst-libs/gst/drm/meson.build
@@ -6,7 +6,7 @@ gstdrm_headers = [
 ]
 install_headers(gstdrm_headers, subdir : 'gstreamer-1.0/gst/drm')
 
-libdrm_dep = dependency('libdrm', version: '>= 2.4.55', required: get_option('wayland'))
+libdrm_dep = dependency('libdrm', version: '>= 2.4.55')
 if libdrm_dep.found()
   gstdrm = library('gstdrm-' + api_version,
     gstdrm_sources,
-- 
2.31.1

